src/App.js

```diff
@@ -1,22 +1,27 @@
-import logo from './logo.svg';
 import './App.css';
+import { React, Fragment } from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import Nav from './components/Nav';
+import Categories from './redux/categories/categories';
+import Booklist from './components/Booklist';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <Router>
+          <>
+            <Nav />
+            <Routes>
+              <Route path="/categories" element={<Categories />} />
+              <Route path="/" element={<Booklist />} />
+            </Routes>
+          </>
+        </Router>
```

Comment:
- [ ] [OPTIONAL] Each route should have the attribute exact since `react-router` paths work with pattern matching(it will match against the value of `path`).

Reply:
Hi, @omar-labana and @Rayan84, because we are all here to learn, let me bring you up to speed with this info:

Since the student is using version 6 of `react-router-dom`, there is no need for the `exact` attribute. You can find more info from this [guide](https://reactrouter.com/docs/en/v6/upgrading/v5) about upgrading from v5 to v6. Just a quote from the guide:

---
> <Route exact> is gone. Instead, routes with descendant routes (defined in other components) use a trailing * in their path to indicate they match deeply